Gemfile.lock

```diff
@@ -142,6 +142,3 @@ DEPENDENCIES
   rubocop
   webmock
   webpack-rails!
-
-BUNDLED WITH
```

Comment:
Unrelated to the content, I'd recommend updating the version of bundler that you use so that it doesn't remove this 'BUNDLED WITH' footer.